commands/Mod/mute.js

```diff
@@ -56,10 +56,10 @@ module.exports = {
   // Configuração do comando
   config: {
     name: "mute",
-    noalias: "Sem sinonimos",
+    noalias: "Sem sinônimos",
     aliases: [],
-    description: "De um mute para um membro do server",
-    usage: "mute <@member> <tempo><m/h/d/y> [motivo]",
+    description: "Silencie um membro do servidor por uma certa quantidade de tempo.",
+    usage: "mute <@membro> <tempo><m/h/d/a> [motivo]",
```

Comment:
Seguir padrão do codigo do bot
```suggestion
    usage: "mute <@membro> <tempo><m/h/d/y> [motivo]",
```